content/zh/docs/ops/setup/webhook/index.md

```diff
@@ -1,60 +1,41 @@
-and do something with them. You can define two types of admission
-webhooks, validating admission webhook and mutating admission
-webhook. With validating admission webhooks, you may reject requests
-to enforce custom admission policies. With mutating admission
-webhooks, you may change requests to enforce custom defaults.
+准入 Webhook 是 HTTP 方式的回调，接收准入请求并对其进行相关操作。可定义两种类型的准入 Webhook，Validating 准入 Webhook 和 Mutating 准入 Webhook。 使用 Validating Webhook，可以通过自定义的准入策略来拒绝请求；使用 Mutating Webhook，可以通过自定义默认值来修改请求。
 {{< /tip >}}
 
-Istio uses `ValidatingAdmissionWebhooks` for validating Istio
-configuration and `MutatingAdmissionWebhooks` for automatically
-injecting the sidecar proxy into user pods.
+Istio 使用 `ValidatingAdmissionWebhooks` 验证 Istio 配置，使用 `MutatingAdmissionWebhooks` 自动将 Sidecar 代理注入至用户 Pod
 
-The webhook setup guides assuming general familiarity with Kubernetes
-Dynamic Admission Webhooks. Consult the [Kubernetes API references](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/) for
-detailed documentation of the mutating and validating webhook configuration.
+Webhook 设置过程需要了解 Kubernetes 动态准入 Webhook 相关的知识。有关 Validating 和 Mutating Webhook 配置的详细文档，请参考 [Kubernetes API](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/)。
 
-## Verify dynamic admission webhook prerequisites
+## 验证动态准入 Webhook 前置条件
 
-See the [platform setup instructions](/docs/setup/platform-setup/)
-for Kubernetes provider specific setup instructions. Webhooks will not
-function properly if the cluster is misconfigured. You can follow
-these steps once the cluster has been configured and dynamic
-webhooks and dependent features are not functioning properly.
 
-1. Verify you’re using a supported version ({{< supported_kubernetes_versions >}}) of
-   [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and of the Kubernetes server:
+请参阅 [平台设置说明](/zh/docs/setup/platform-setup/)。如果群集配置错误，Webhook 将无法正常工作。集群配置后，当动态 Webhook 和相关特性不能正常工作时，你可以通过以下步骤进行检查。
+
+1. 验证当前是否使用正确版本的 [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 和 kubernetes 服务
 
     {{< text bash >}}
     $ kubectl version --short
     Client Version: v1.10.2
     Server Version: v1.10.4-gke.0
     {{< /text >}}
 
-1. `admissionregistration.k8s.io/v1beta1` should be enabled
+2. `admissionregistration.k8s.io/v1beta1` 应是启用状态
 
     {{< text bash >}}
     $ kubectl api-versions | grep admissionregistration.k8s.io/v1beta1
     admissionregistration.k8s.io/v1beta1
     {{< /text >}}
 
-1. Verify `MutatingAdmissionWebhook` and `ValidatingAdmissionWebhook` plugins are
-   listed in the `kube-apiserver --enable-admission-plugins`. Access
-   to this flag is [provider specific](/docs/setup/platform-setup/).
+3. 验证 `kube-apiserver --enable-admission-plugins` 配置中插件 `MutatingAdmissionWebhook` 和 `ValidatingAdmissionWebhook` 在是否被启用。通过检查[指定规范](/zh/docs/setup/install/kubernetes/#prerequisites)中的标志（`--enable-admission-plugins`）。
```

Comment:
```suggestion
3. 验证 `kube-apiserver --enable-admission-plugins` 配置中插件 `MutatingAdmissionWebhook` 和 `ValidatingAdmissionWebhook` 是否被启用。通过检查[指定规范](/zh/docs/setup/install/kubernetes/#prerequisites)中的标志（`--enable-admission-plugins`）。
```

Reply:
3->1

Reply:
/zh/docs/setup/install/kubernetes/#prerequisites  this url is wrong. see en/ origin files